geometry_msgs/Pose drone_pose
sensor_msgs/PointCloud2[] drone_hulls
drone_colors hulls_colors
int32[] largest_hulls
drone_ver[] v_hulls